packages/wdio-appium-service/tests/launcher.test.ts

```diff
@@ -102,11 +105,10 @@ describe('Appium launcher', () => {
                 args: { foo: 'bar' }
             }
             const capabilities = {
-                browserA: { port: 1234 },
-                browserB: {}
+                browserA: { port: 1234 } as WebDriver.DesiredCapabilities,
+                browserB: {} as WebDriver.DesiredCapabilities
             }
             const launcher = new AppiumLauncher(options, capabilities, {})
-            launcher._startAppium = jest.fn().mockImplementation((cmd, args, cb) => cb(null, new MockProcess()))
             await launcher.onPrepare()
             expect(capabilities.browserA.protocol).toBe('http')
```

Comment:
these properties in test are being highlighted by Typescript linter as non existent property, even though I extended WebDriver DesiredCapabilities in `appiumservice.d.ts`. 
Tests has passed, so I'll address this warning in another PR, so we can merge this one.